versionsupport_common/src/main/java/com/tomkeuper/bedwars/utils/MainUtils.java

```diff
@@ -0,0 +1,138 @@
+package com.tomkeuper.bedwars.utils;
+
+import com.tomkeuper.bedwars.api.arena.GameState;
+import com.tomkeuper.bedwars.api.arena.IArena;
+import com.tomkeuper.bedwars.api.events.player.PlayerGeneratorCollectEvent;
+import com.tomkeuper.bedwars.api.server.ServerType;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityPickupItemEvent;
+import org.bukkit.event.player.PlayerPickupItemEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.tomkeuper.bedwars.support.version.common.VersionCommon.api;
+
+public class MainUtils {
+    public static List<Item> getItemsAround(Item item) {
+        List<Item> items = item.getNearbyEntities(0.5, 0.5, 0.5).stream()
+                .filter(entity -> entity instanceof Item)
+                .filter(entity -> ((Item) entity).getItemStack().getType() == item.getItemStack().getType())
+                .filter(entity -> entity.getLocation().distance(item.getLocation()) < 0.1)
+                .map(entity -> (Item) entity)
+                .collect(Collectors.toList());
+        items.add(item);
+        return items;
+    }
+
+    public static void manageGeneratorPickUp(PlayerPickupItemEvent e, Player player, Item item, List<Item> items) {
+        int am = items.size();
```

Comment:
int am? what does 'am' mean? O-o

Reply:
amount, i changed it because i tried doing something before that and forgot to rename it back again, mb